kong.conf.default

```diff
@@ -345,6 +345,13 @@
                                  # - certificate content
                                  # - base64 encoded certificate content
 
+#cluster_rpc =                   # Enable/disable cluster rpc framework.
+                                 # The default value is `off`.
```

Comment:
```suggestion
#cluster_rpc = off               # The setting to enable or disable RPC-based 
                                 # communication between control plane and data plane nodes.
                                 # The valid values are `on` and `off.` Set to `on` to enable.
                                 # If `cluster_rpc_sync` is set to `on`, this parameter 
                                 # must also be set to `on`.
```
I used Euclid's suggestion and rephrased it for readability.

You don't need to say what the default value is in the description, that goes next to the parameter itself.

Reply:
done, thanks.